proxy/ProxyTransaction.h

```diff
@@ -37,7 +37,7 @@ class ProxyTransaction : public VConnection
 
   /// Virtual Methods
   //
-  virtual void new_transaction();
+  virtual void new_transaction(bool from_early_data = false);
```

Comment:
Looks good to me. H3/QUIC doesn't have this protection but I think this interface works for H3 too.